openedx/features/ucsd_features/tests/tests.py

```diff
@@ -1,59 +0,0 @@
-import json
```

Comment:
Reason: These tests were written for custom contact_us view. Now we do not need it.